Library/Formula/osxfuse.rb

```diff
@@ -25,6 +25,9 @@ class Osxfuse < Formula
 
   conflicts_with "fuse4x", :because => "both install `fuse.pc`"
 
+  # allow building with Xcode 6.3
+  patch :DATA
```

Comment:
@mikemcquaid They've committed support, is using that revision (as I've done) instead of a tag an acceptable temporary solution? That's the only commit that's happened in the `osxfuse-2` branch since the 2.7.5 tag.